unit-tests/src/test/java/org/eclipse/collections/impl/utility/ListIterateTest.java

```diff
@@ -411,6 +621,25 @@ private void assertReverseForEachWithIndex(List<Integer> list)
         });
     }
 
+    @Test
+    public void forEach()
```

Comment:
Rename the test `forEachInRange`, even though the method is an overloaded of `forEach`.

Reply:
Done.